src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -134,6 +136,22 @@ default String getAsTextShowFee() {
         return builder.toString();
     }
 
+    /**
+     * Formats the person as text, showing name and assessments + grades.
+     */
+    default String getAsTextShowAssess() {
+        final StringBuilder builder = new StringBuilder();
+        final String stringChain = Formatter.getPrintableString(
+                true,
+                getName());
```

Comment:
Er, maybe making this inline may make it easier to read as compared to new-lined